### ISSUE TITLE
Update Riders Facing Comms Plan for Launch

### ISSUE THREAD

**Issue by vykster** [cal-itp/benefits#650]
Todo List:
**RIDER Facing**

- [x] Update draft MST website copy to include login.gov information: [MST Website /benefits page update](https://docs.google.com/document/d/1I2SBHlMViGgPPC84C7gxlimZ4lXYH1JR/edit) (mst.org/benefits)
- [x] Draft email to riders
- [x] https://github.com/cal-itp/benefits/issues/645
 - [ ] (test if mobile wallets are working/ update cust serv doc if working)
- [x] Consider new press releases for MST including login.gov content
- [x] Review from CDT D-ID team, addition of additional messaging, content, or channels to support team’s goals.
- [ ] Coordinate with payments team to verify bank card info is up to date (ie: if amex is ready)
- [x] Create new images for [social media posts](https://docs.google.com/document/d/1WCZeRx6FN82c8snqXzIvPvyM-oalryieeKsblxEjzWA/edit#heading=h.hf639nip271v) announcing launch.
- [x] Create image(s) for user recruiting [social media posts](https://docs.google.com/document/d/1WCZeRx6FN82c8snqXzIvPvyM-oalryieeKsblxEjzWA/edit#heading=h.hf639nip271v). 

**Other**
Split non-rider facing comms into new issue. #896 

Resources:
- [Cal-ITP Benefits Launch Comms Plan](https://docs.google.com/document/d/11wfslytJJ02zb-u8fsY4D92CiG8EU9DXWUENeqnPMtY/edit#heading=h.ecujg6dssk7t)

**Comment by afeld** [comment]
Idea for a technical blog post (assuming this is the right issue to post it in): setting up logging and error monitoring for Django in Azure App Service.

**Comment by afeld** [comment]
Left comments/suggestions on all the [rider assets](https://docs.google.com/document/d/11wfslytJJ02zb-u8fsY4D92CiG8EU9DXWUENeqnPMtY/edit#heading=h.gknovl9qvvsm). Google Slides doesn't support suggested edits; let me know if you're ok with me modifying the [explanatory deck](https://docs.google.com/presentation/d/1Vawi8d70jtKfL-u5lwM05CcDyQ5zB764/edit#slide=id.p1) directly (to resolve the comments).

**Comment by vykster** [comment]
@afeld yes go for it please.

**Comment by thekaveman** [comment]
@vykster is there more to do here or can we close this out?

**Comment by vykster** [comment]
I think we do just want to verify with the payments team that there have been no changes to the payment types accepted since last launch. 

@esquared415 can you confirm that the payments info is up to date at https://benefits.calitp.org/help, or direct us to the someone else?

**Comment by esquared415** [comment]
Thanks for flagging me to this query @afeld (and apologies for not seeing it earlier, @vykster).

@jenro1, is there an eta for when Discover and AmEx will be certified on our MSA-awarded validators / payment acceptance devices? (The Benefits dev team is wondering how much longer the accepted payment methods will be just contactless Visa and Mastercard credit and debit cards.) Thanks!

**Comment by jenro1** [comment]
Hi all - no changes to accepted payment types at this time. 

Discover is currently working on their L3 certification with Cybersource, LP, SC Soft and Kuba. They're hoping to get a "slot" for that in October. I would need to circle back with them to understand end to end timelines. We're waiting on an update from Amex and speaking with them again in 2 weeks' time so I can provide more information then.

**Comment by afeld** [comment]
Sounds like the page is up-to-date for now. We can update the payment info if/when that changes. Thanks!
